test/e2e/packagemanifest_e2e_test.go

```diff
@@ -36,9 +36,7 @@ var _ = Describe("Package Manifest API lists available Operators from Catalog So
 		pmc = newPMClient()
 		c = ctx.Ctx().KubeClient()
 
-		BeforeEach(func() {
-			generatedNamespace = SetupGeneratedTestNamespace(genName("package-manifest-e2e"))
-		})
+		generatedNamespace = SetupGeneratedTestNamespace(genName("package-manifest-e2e"))
```

Comment:
Why do we want one ns for all of them vs one per test?

Reply:
Hmm I might be missing what you're trying to point out, could you describe what you mean by "all of them" vs "one per test"? Running the tests in this describe block shows that multiple namespaces are generated, specifically one for each test.

```
$ k get ns -w 
...
...
package-manifest-e2ect2wt   Active   0s
package-manifest-e2ehsr64   Active   0s
package-manifest-e2erptpp   Active   0s
package-manifest-e2edgq5m   Active   0s
```

Reply:
Silly Pseudo Code used to explain what is happening:
```
Describe("e2e suite")...{
   BeforeEach{
      fmt.Printf("Before\n)
   }
   AfterEach{
      fmt.Printf("After\n)
   }
   It(1)...{
      fmt.Printf("Test 1\n)
   }
   It(2)...{
      fmt.Printf("Test 2\n)
   }
}
```

If you run the tests above, you'd see:
1. Before
2. Test1
3. After
4. Before
5. Test2
6. After